SConstruct

```diff
@@ -1,515 +0,0 @@
-import os, sys
```

Comment:
This should at least go in a separate commit so that we can easily decide when to delete SCons, but as mentioned - just leave both SCons and meson as is and we can decide when to remove them afterwards.